nusantara/nusa_datasets/id_google_play_review/id_google_play_review.py

```diff
@@ -0,0 +1,156 @@
+            schema="nusantara_text",
+            subset_id="id_google_play_review_posneg",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "id_google_play_review_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+
+        # Create the source schema; this schema will keep all keys/information/labels as close to the original dataset
+        # as possible.
+
+        # You can arbitrarily nest lists and dictionaries.
+        # For iterables, use lists over tuples or `datasets.Sequence`
+
+        if self.config.schema == "source":
+            features = datasets.Features({
+                "text": datasets.Value("string"),
+                "label": datasets.Value("string"),
+                "stars": datasets.Value("int8")
+            })
+        elif self.config.schema == "nusantara_text":
+            if self.config.subset_id == "id_google_play_review_posneg":
+                features = schemas.text_features(["pos", "neg"])
+            elif self.config.subset_id == "id_google_play_review":
+                features = schemas.text_features(["1", "2", "3", "4", "5"])
+            else:
+                raise ValueError(f"Invalid config: {self.config.name}")
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+        urls = _URLS[_DATASETNAME]
+        train_data_path = Path(dl_manager.download(urls["train"]))
+        valid_data_path = Path(dl_manager.download(urls["valid"]))
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={"filepath": train_data_path, "split": "train"},
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.VALIDATION,
+                gen_kwargs={"filepath": valid_data_path, "split": "valid"},
+            ),
+        ]
+
+    def _generate_examples(self, filepath: Path, split: str) -> Tuple[int, Dict]:
+        """Yields examples as (key, example) tuples."""
+
+        df = pd.read_csv(filepath, sep=",").reset_index()
+        for row in df.itertuples(index=False):
+            if self.config.schema == "source":
+                example = {"text": row.text, "label": row.label, "stars": row.stars}
+                yield row.index, example
+            elif self.config.schema == "nusantara_text":
+                if self.config.subset_id == "id_google_play_review_posneg":
+                    example = {"id": row.index, "text": row.text, "label": row.label}
+                elif self.config.subset_id == "id_google_play_review":
+                    example = {"id": row.index, "text": row.text, "label": str(row.stars)}
```

Comment:
Could you maybe tell why should the row.stars in this line be str by the way? or should it be just int?

Reply:
@bryanwilie 
Thank you for reviewing.
1. Noted on the citation. 
2. About the row.stars being converted to string, my understanding was that the `ClassLabel` accepts `names=label_names` parameter, which is defined as list of label strings. I think it would be better to keep this as int, similar to the one used for the `source`. Though this would require changing the `label_names` to [0, 1, 2, 3, 4, 5] to run well (the label is only 1 to 5, adding 0 would be out of place, IMO). Would it be better to change this to int?

Reply:
ah yes you're correct. I think it'll be fine this way. Thanks @jen-santoso!